cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AuthorizationHandler.java

```diff
@@ -83,7 +83,7 @@ public class AuthorizationHandler extends AbstractAppFabricHttpHandler {
   private final boolean authenticationEnabled;
   private final boolean authorizationEnabled;
   private final PermissionManager permissionManager;
-  private final AccessController accessController;
+  private final AccessControllerV2 accessController;
```

Comment:
I don't think we should expose an SPI directly through a public API. It should still use API, so that we do proper audit logging

Reply:
This would require adding support for CAL for all those calls though. And that's a right thing to do even if we don't provide it for IAM

Reply:
By API here do you that we should use `AccessEnforcer` implemented interface here ? 
meaning old `AccessController` ? or create a new one ?